src/library_webgpu.js

```diff
@@ -399,6 +399,11 @@ var LibraryWebGPU = {
       'device-lost',
       'unknown',
     ],
+    CompilationMessageType: [
+      'error',
+      'warning',
+      'info',
+    ],
```

Comment:
We can map this in reverse (string-to-number instead of number-to-string), similar to BufferMapState. Then you can look up the index directly instead of using `indexOf`.